### ISSUE TITLE
Task/Add A11y check to all e2e tests

### ISSUE THREAD

**Issue by Sepehr-Sobhani** [bcgov/cas-registration#4584]
#### Describe the task

We have `analyzeAccessibility` check in some e2e tests, but we should add it to all tests to ensure all pages follow A11y rules.
Every page in the app that has an E2E test should be analyzed using the `analyzeAccessibility` function. There are plenty of existing examples in the codebase that can be helpful to look at. For instance: https://github.com/bcgov/cas-registration/blob/0250ca6374164bab9d3cd04427ac40afd4065b9c/bciers/apps/administration-e2e/tests/workflows/contacts.spec.ts#L68

#### Acceptance Criteria

- [ ] All e2e tests have one instance of `analyzeAccessibility`

#### Definition of Done

- [ ] All pages are analyzed using `analyzeAccessibility`
- [ ] All issues identified by this function are resolved

**Comment by patriciarussellCAS** [comment]
Note: Split into separate tickets for each module: Admin/Reg, Reporting, Compliance

**Comment by zoeyli-46** [comment]
Refinement notes: can be pushed because MUI packages should be updated first #4634
